pages/windows/where.md

```diff
@@ -16,6 +16,6 @@
 
 `where /R {{path/to/directory}} {{file_pattern}}`
 
-- Display only the error code for the location of file pattern:
+- Only return the error code for the location of file pattern:
```

Comment:
```suggestion
- Silently return the error code for the location of the file pattern:
```
@kbdharun 
What if we change it this way to reflect the fact that nothing is displayed, and the other command also return error code?